html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/broadcastchannel-success.html

```diff
@@ -30,7 +30,7 @@
           return;
         }
 
-        assert_true(i === 0 || i === 1 || i === 2, `Any message events must come from expected sources; got ${i}`);
+        assert_in_array(i, [0, 1, 2], "Any message events must come from expected sources");
```

Comment:
I presume the actual value was provided in the message because the assert didn't provide it (i.e., if it failed it would just have output "got false, expected true") whereas now it does.